shapes/README.md

```diff
@@ -0,0 +1,54 @@
+# Shapes
+
+We'll practice list comprehensions in this exercise by printing different shapes out given their sizes.
+
+```elixir
+iex> Shapes.square(5)
+* * * * *
+* * * * *
+* * * * *
+* * * * *
+* * * * *
+
+iex> Shapes.rectangle(5, 3)
+* * *
+*   *
+*   *
+*   *
+* * *
+
+iex> Shapes.rectangle(4, 8)
+* * * * * * * *
+*             *
+*             *
+* * * * * * * *
+
+iex> Shapes.pyramid(5)
+    *
+   * *
+  * * *
+ * * * *
+* * * * *
+
+iex> Shapes.pyramid(5)
```

Comment:
It seems like it's `diamond` not `pyramid`. And in [the `diamond's` `@doc` ](https://github.com/elixirschool/homework/pull/5/files#diff-0c9166d8d48d794f06d9a1c7189870e5R50) too.